.github/workflows/deploy.yaml

```diff
@@ -0,0 +1,57 @@
+name: Build
+
+on:
+  workflow_dispatch:
```

Comment:
This is for manual triggers - it could be switched to build on new changes push to master e.g